repository/jsrepository.json

```diff
@@ -2212,6 +2217,19 @@
 				"info": [
 					"https://github.com/ckeditor/ckeditor4/security/advisories/GHSA-f6rf-9m92-x2hh"
 				]
+			},
+			{
+				"below" : "4.21.0",
+				"atOrAbove" : "4.19.1",
```

Comment:
Where did you find 4.19.1 ? https://github.com/ckeditor/ckeditor4/security/advisories/GHSA-vh5c-xwqv-cv9g only mentions below 4.21.0